tikv-configuration-file.md

```diff
@@ -36,6 +36,17 @@ This document only describes the parameters that are not included in command-lin
 + The threshold for outputing slow logs. If the processing time is longer than this threshold, slow logs are output.
 + Default value: `"1s"`
 
+### `memory-usage-limit`
+
++ The memory usage limit for the TiKV instance. When the memory usage of TiKV almost reaches this threshold, it can occupy some internal components (such as evicting cached Raft entries) to release memory.
```

Comment:
+ What does "almost" mean? It means the memory usage does not reach this threshold.
+ "occupy some internal components"? Does it mean "占用一些内部组件"？or "occupy the memory of some internal components"?
```suggestion
+ The limit on memory usage of the TiKV instance. When the memory usage of TiKV almost reaches this threshold, it can occupy some internal components (such as evicting cached Raft entries) to release memory.
```

Reply:
"almost" means the action will be triggered before the usage actually reaches the threshold.